src/note/asset_base.rs

```diff
@@ -54,21 +54,30 @@ impl AssetBase {
     ///
     /// # Panics
     ///
-    /// Panics if `asset_desc` is empty or greater than `MAX_ASSET_DESCRIPTION_SIZE`.
+    /// Panics if `asset_desc` is empty or greater than `MAX_ASSET_DESCRIPTION_SIZE` or if the derived Asset Base is the identity point (this will happen with negligible probability).
```

Comment:
move `(this will happen with negligible probability)` as a comment above the assert

Reply:
Done.